client/activities/activity-back-button.tsx

```diff
@@ -22,6 +24,10 @@ export function ActivityBackButton() {
   }, [dispatch])
 
   return shouldShow ? (
-    <BackButton icon={<MaterialIcon icon='arrow_back' />} title='Back' onClick={onBackClick} />
+    <BackButton
+      icon={<MaterialIcon icon='arrow_back' />}
+      title={t('common.literals.back', 'Back')}
```

Comment:
nit: No idea what 'literals' is really meant to contain in this context. Maybe it should be `common.actions`?

Reply:
Yeah, not sure. I already added the `common.literals.X` key during the auth stuff, mostly for some very common strings, but not sure if this organization makes much sense: https://github.com/ShieldBattery/ShieldBattery/blob/e411f812dc80ad535d72ef893ebf8ffd645c758b/server/public/locales/en/global.json#L114-L124

Reply:
I do not think it really makes sense, no. There are some things in here that are pretty common (Cancel, Continue, Required, Save), but there are bunch of things that are auth-specific and should be in the auth stuff, not some random common stuff. `literals` has no meaning here, "string literal" in programming just means "a string that is directly in a code, not a variable" so I don't really see what that can mean in this context.

Other nit: "login" is a bad naming because the string is "Log in", it should be "logIn"

Reply:
Yeah, makes sense. I removed the `common.literals.X` key, and instead used `common.actions.X` for a few of those common actions. For all other stuff I just used the key that was already used for that file.

Reply:
Sounds good 👍